package.json

```diff
@@ -63,6 +63,7 @@
     "artillery": "^1.6.0-28",
     "await-mutex": "^1.0.2",
     "axios": "^0.19.0",
+    "big-number": "^2.0.0",
```

Comment:
BN is already use all over. look for examples we dont need to import this package

Reply:
done